### ISSUE TITLE
Feature Request: Ability to specify a list of snapshot extensions

### ISSUE THREAD

**Issue by dawnmist** [asvetliakov/typescript-snapshots-plugin#4]
When using [Storybook](https://github.com/storybooks/storybook) with the [Storyshots](https://github.com/storybooks/storybook/tree/master/addons/storyshots) addon, stories are turned into Jest snapshot tests with a snapshot file extension of '.storyshot'. The reason for them using different suffix is so that storyshots is able to identify orphaned snapshot files that came from their plugin rather than the normal jest tests.

As a result, we have both '.snap' and '.storyshot' snapshot files. It would be nice to be able to configure the plugin to identify the '.storyshot' files as also being snapshot files.

**Comment by asvetliakov** [comment]
@dawnmist 
So look into both '.snap' and '.storyshot' files (i.e. your test file can have both) or only into single one (either the '.snap' or '.storyshot')?

**Comment by dawnmist** [comment]
At this stage:
* If the test file is a '*.stories.tsx?' file, it has "*.snapshot" snapshots.
* Normal test files have the normal '*.snap' snapshots.

However, being able to look into both allows for that to change in future, and there is another addon for Storyshots that can potentially cause both to be in the same file (the Specs addon).

If it was made into a configuration option that takes an array of strings for the extension, with a default value of `["*.snap"] ` it would work out of the box for the majority of people while being able to be extended to add additional extensions for people using something like Storybook.

**Comment by asvetliakov** [comment]
Just published 1.2 version, you need to configure:
```json
            {
                "name": "typescript-snapshots-plugin",
                "snapshotFileExtensions": [".storyshot", ".snap"]
            },
```
then restart the ts server/editor. Feel free to reopen this issue if you'll have any problems
